recipes/default.rb

```diff
@@ -16,9 +16,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-include_recipe 'apt::default'
-include_recipe 'ark::default'
+include_recipe 'mcrouter::_user'
+include_recipe 'mcrouter::_deps_centos' if node['platform'] == 'centos'
+include_recipe 'mcrouter::_deps_ubuntu_16.04' if node['platform'] == 'ubuntu' && node['platform_version'].to_f	== 16.04
 include_recipe 'memcached::default' if node['mcrouter']['local_memcached']
+include_recipe 'mcrouter::_double_conversion'
+include_recipe 'mcrouter::folly'
```

Comment:
Same thing for `folly` and `wangle` as for `double_conversion` : are they intended to be allowed to added in a run-list, if not should have leading underscore, otherwise might need recipes updated to ensure things like `autoreconf` exist on $PATH.

Although, that being said, looks like some of those potential problems previously existed ...